front/pages/api/w/[wId]/data_sources/managed.ts

```diff
@@ -193,6 +193,16 @@ async function handler(
           }
           dataSourceDescription = configurationRes.value.url;
           break;
+        case "slack":
+          // When creating a Slack data source we don't receive a configuration but pass a default
```

Comment:
`nit` feels weird to have this "default initialisation" part being buried in all this code.

Reply:
Well it's next to the init of webcrawler and unfortunately no better place to put it. We surely don't want it buried client side and ideally we would just not pass it but the type system as currently designed requires it :/